digdag-tests/src/test/java/acceptance/td/TdResultExportIT.java

```diff
@@ -0,0 +1,205 @@
+import io.netty.handler.codec.http.FullHttpRequest;
+import okhttp3.MediaType;
+import okhttp3.OkHttpClient;
+import okhttp3.Request;
+import okhttp3.RequestBody;
+import okhttp3.Response;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.TemporaryFolder;
+import org.littleshoot.proxy.HttpProxyServer;
+import utils.TemporaryDigdagServer;
+import utils.TestUtils;
+
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.nio.file.Path;
+import java.time.Duration;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.UUID;
+import java.util.concurrent.Callable;
+
+import static acceptance.td.Secrets.TD_API_KEY;
+import static acceptance.td.Secrets.TD_API_ENDPOINT;
+import static org.hamcrest.Matchers.is;
+import static org.hamcrest.Matchers.isEmptyOrNullString;
+import static org.hamcrest.Matchers.not;
+import static org.junit.Assert.assertThat;
+import static org.junit.Assert.fail;
+import static org.junit.Assume.assumeThat;
+import static utils.TestUtils.addWorkflow;
+import static utils.TestUtils.attemptSuccess;
+import static utils.TestUtils.copyResource;
+import static utils.TestUtils.expect;
+import static utils.TestUtils.objectMapper;
+import static utils.TestUtils.pushAndStart;
+import static utils.TestUtils.startRequestTrackingProxy;
+
+public class TdResultExportIT
+{
+    @Rule
+    public TemporaryFolder folder = new TemporaryFolder();
+
+    private TDClient client;
+    private String database;
+    private String table;
+    private String connectionId;
+    private static final MediaType JSON = MediaType.parse("application/json; charset=utf-8");
+    private OkHttpClient httpClient;
+    private Path projectDir;
+
+    public TemporaryDigdagServer server;
+
+    private HttpProxyServer proxyServer;
+    private String sampleJobId;
+
+    @Before
+    public void setUp()
+            throws Exception
+    {
+        assumeThat(TD_API_KEY, not(isEmptyOrNullString()));
+        assumeThat(TD_API_ENDPOINT, not(isEmptyOrNullString()));
+        projectDir = folder.getRoot().toPath().toAbsolutePath().normalize();
+
+        client = TDClient.newBuilder(false)
+                .setApiKey(TD_API_KEY)
+                .setEndpoint(TD_API_ENDPOINT)
+                .build();
+        sampleJobId = client.submit(TDJobRequest.newPrestoQuery("sample_datasets",
+                "select time from www_access"));
+        database = "_digdag_integration_result_export_td_test_db";
+        client.createDatabaseIfNotExists(database);
+        table = "_digdag_integration_result_export_td_test_table";
+        client.createTableIfNotExists(database, table);
+        this.httpClient = new OkHttpClient();
+    }
+
+    @Test
+    public void testSubmitResultExportJob()
+            throws Exception
+    {
+        String resultConnectorName =  "test_" + UUID.randomUUID().toString().replace('-', '_');
```

Comment:
Very minor things. Name prefix 'digdag_test_' may be better to make it clear.

Reply:
I fixed here.
https://github.com/treasure-data/digdag/pull/1240/commits/505e0f092a0aad91606ab44b5606519c0dc90c09